File: examples/Altcha.Net.Examples.AspNetMvc.CSharp/AltchaProvider.cs
using System;
using Altcha.Net;

namespace Altcha.Net.Examples.AspNetMvc.CSharp
{
    public static class AltchaProvider
    {
        public static readonly AltchaService Service = new AltchaService(
            new AltchaOptions
            {
                SecretKey = Environment.GetEnvironmentVariable("ALTCHA_SECRET"),
                ChallengeExpiry = TimeSpan.FromMinutes(2),
                Complexity = new AltchaComplexity(50000, 100000)
            },
            new MemoryAltchaReplayStore());
    }
}

File: examples/Altcha.Net.Examples.AspNetWebForms.VbNet/AltchaProvider.vb
Imports Altcha.Net

Public NotInheritable Class AltchaProvider
    Private Sub New()
    End Sub

    Public Shared ReadOnly Service As New AltchaService(
        New AltchaOptions With {
            .SecretKey = Environment.GetEnvironmentVariable("ALTCHA_SECRET"),
            .ChallengeExpiry = TimeSpan.FromMinutes(2),
            .Complexity = New AltchaComplexity(50000, 100000)
        },
        New MemoryAltchaReplayStore())
End Class
